prjxray/util.py

```diff
@@ -172,43 +172,34 @@ def addr2btype(base_addr):
     return block_type_i2s[block_type_i]
 
 
-def gen_tile_bits(db_root, tilej, strict=False, verbose=False):
+def gen_tile_bits(tile_segbits, tile, strict=False, verbose=False):
```

Comment:
Move gen_tile_bits out of prjxray/util.py, it is only used by checkdb.py

Reply:
Ok, makes sense